mmdet3d/models/dense_heads/vote_head.py

```diff
@@ -318,6 +324,25 @@ def loss(self,
             size_class_loss=size_class_loss,
             size_res_loss=size_res_loss)
 
+        if self.iou_loss:
+            mean_sizes = size_res_targets.new_tensor(
+                self.bbox_coder.mean_sizes)[None]
+            mean_sizes = torch.sum(mean_sizes * one_hot_size_targets_expand, 2)
+            size_pred = (size_residual_norm + 1) * mean_sizes
+            size_target = (size_res_targets + 1) * mean_sizes
+            center_pred = bbox_preds['center']
+            size_pred = torch.clamp(size_pred, 0)
+            pred = torch.cat(
+                [center_pred - size_pred / 2, center_pred + size_pred / 2],
+                dim=-1)
+            target = torch.cat([
+                assigned_center_targets - size_target / 2,
+                assigned_center_targets + size_target / 2
+            ],
+                               dim=-1)
```

Comment:
Seems a little bit strange? Is it the result after pre-commit fix?

Reply:
Yes, yapf did this. I managed to avoid this.